### PR TITLE
Added Animated Gradient Border Button

### PR DESCRIPTION
**Pull request by parresh77**
Issue: #418

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -154,6 +154,20 @@ $(document).ready(function () {
       label: "Animated Buttons",
       classes: "",
       children: [
+        {
+          type: "gradient-border",
+          label: "Animated Gradient Border",
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:159]: remove the animated from the label and place it then at the correct alphabetical order.

File: src/components/animated/_gradient-border.less [hunk 2]
```diff
@@ -0,0 +1,53 @@
+@import "https://fonts.googleapis.com/css?family=Raleway:200";
```
Comment (shahednasser) [inline src/components/animated/_gradient-border.less:1]: please remove font as it is not necessary for the design of the button

**Review by shahednasser — CHANGES_REQUESTED** [review]
I like your work but I have a question: why are there no rounded and block button? I get it might not work for rounded but please at least make a normal sized button and a block button.

**Comment by parresh77** [comment]
@shahednasser I've made all the requested changes. I've also changed the dimensions of the button to make it look good collectively in both basic and block size.

File: assets/js/buttons-examples.js [hunk 3]
```diff
@@ -211,6 +211,20 @@ $(document).ready(function () {
           label: "Flash",
           classes: "flash-btn",
         },
+        {
+          type: "gradient-border",
+          label: "Gradient Border",
+          classes: "gradient-border",
+          text: "Button",
+          shouldHaveRoundedType: false,
+          // shouldHaveBlockType: false,
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:220]: Remove this line

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by parresh77** [comment]
@shahednasser Removed the comment.

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @parresh77 for code

**Pull request merged**